include/bout/mesh.hxx

```diff
@@ -568,8 +568,9 @@ class Mesh {
    */
   void setParallelTransform();
 
-    //Region related routines
-
+  //Region related routines
+  int maxregionblocksize;
```

Comment:
Could do with a short doxygen comment here, and some separation from the existing comment